### ISSUE TITLE
Universes Within (SLX) and the corresponding Secret Lair cards aren't on Gatherer

### ISSUE THREAD

**Issue by spacemonaut** [scryfall/gatherer-bugs#139]
In October 2021, the [_Stranger Things_ Secret Lair][strangerthings] was released, and in _March 2022_, the same cards were re-released in [Universes Within (SLX)][slx]. You can see [all those cards here][all].

However, no version of any of these cards exists on Gatherer. That's even true for Universes Within, which doesn't use third-party licensed material.

* There are no cards with the text “friends forever” on Gatherer: https://gatherer.wizards.com/Pages/Search/Default.aspx?action=advanced&text=+[%22friends%20forever%22]
* Neither Arvinox nor Mind Flayer exists: https://gatherer.wizards.com/Pages/Search/Default.aspx?action=advanced&type=+[%22Legendary%22]&name=|[arvinox]|[%22mind%20flayer%22]
* Neither Havengul Laboratory nor Hawkins National Laboratory exist: https://gatherer.wizards.com/Pages/Search/Default.aspx?action=advanced&name=|[%22Havengul%20Laboratory%22]|[%22hawkins%22]

Finally, there is no “Universes Within” set available to search by in the set dropdown. It ought to come alphabetically between “Unhinged” and “Unlimited Edition”.

![2022-11-13_12-59-05_firefox](https://user-images.githubusercontent.com/4585677/201522876-ee50f942-892b-41e4-be42-d4f5f19ef370.png)

[strangerthings]: https://scryfall.com/search?order=set&q=set%3Asld+%28%28cn%3E%3D340+cn%3C%3D348%29+or+cn%3A609%29&unique=prints
[slx]: https://scryfall.com/sets/slx?as=grid&order=set
[all]: https://scryfall.com/search?q=unique%3Aprints+%28set%3Aslx+or+set%3Asld+in%3Aslx%29

**Comment by spacemonaut** [comment]
This is true for the Street Fighter cards too. Let's take a look at [Zangief / Maarika](https://scryfall.com/search?q=%2B%2B+maarika&unique=cards&as=grid&order=name) for example: Gatherer has no card named [Zangief](https://gatherer.wizards.com/Pages/Search/Default.aspx?name=+[zangief]) and no card named [Maarika](https://gatherer.wizards.com/Pages/Search/Default.aspx?name=+[Maarika]).

**Comment by RivalRowan** [comment]
This issue is caused by the wider issue of _Secret Lair_ cards (CN>250 excl. CN=581) and _The List_ cards not being recorded in Gatherer. It is still present as at September 2, 2025.

Since this ticket was originally opened, only one SLX card has been printed in a product other than a _Secret Lair_ drop or in _The List_ - [Arvinox, the Mind Flail](https://scryfall.com/card/dsc/130/arvinox-the-mind-flail) (SLX version of [Mind Flayer, the Shadow](https://scryfall.com/card/sld/340/arvinox-the-mind-flail)) in _Duskmourn: House of Horror Comander_ (DSC). Since DSC is a product recorded in Gatherer and the SLD/The List cards are not, the DSC version is the _only_ print on Gatherer.

https://gatherer.wizards.com/prints/arvinox-the-mind-flail/en-us?display=cardslarge&page=1
<img width="1344" height="674" alt="Arvinox, the Mind Flail (Duskmourn Commander)" src="https://github.com/user-attachments/assets/48b97927-a565-41fd-a296-60b5c9d301f9" />


The other SLX cards remain missing.
https://gatherer.wizards.com/search?setName=eq~Secret_Lair_Drop~SLD
<img width="883" height="877" alt="Gatherer Advanced Search for SLX." src="https://github.com/user-attachments/assets/cd371b42-1b70-4e3e-af7b-6817f1fb0a10" />
